ccd-definition/CaseEvent/User/UserEventsLRspec-R2-CUI-nonprod.json

```diff
@@ -36,5 +36,21 @@
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0,
     "RetriesTimeoutURLSubmittedEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "INVALID_HWF_REFERENCE",
+    "Name": "Invalid HwF Reference",
+    "Description": "Invalid HwF Reference",
+    "PreConditionState(s)": "*",
```

Comment:
Add the status for pre-condition, I think they are PENDING_CASE_ISSUED and HEARING_READINESS

Reply:
done